modules/apps/portal-vulcan/portal-vulcan-test/src/testIntegration/java/com/liferay/portal/vulcan/internal/graphql/servlet/test/GraphQLServletTest.java

```diff
@@ -589,6 +611,11 @@ public TestDTOPage testDTOPage(
 			return new TestDTOPage(page, pageSize);
 		}
 
+		@com.liferay.portal.vulcan.graphql.annotation.GraphQLField
+		public TestDTO testNonauthorizedUser() throws SecurityException {
```

Comment:
If you type: `git grep testNonaut` there is no pattern in Liferay, only yours. It's better to follow the same pattern, you can use `testUnauthorizedUser`